server/preprocessing/other-scripts/vis_layout.R

```diff
@@ -18,130 +18,189 @@ vis_layout <- function(text, metadata, max_clusters=15, maxit=500, mindim=2, max
   print("calc matrix")
   result <- create_tdm_matrix(metadata, text);
   metadata_full_subjects = result$metadata_full_subjects
-  
+  metadata_full_subjects_tfidf_top5 <- create_node_names(metadata_full_subjects)
```

Comment:
The function `create_node_names` is not defined. Did you mean `create_cluster_names`?